### ISSUE TITLE
Hello, How to set Emacs mode as default mode?

### ISSUE THREAD

**Issue by yestab123** [hlissner/doom-emacs#1338]
1. First open doom emacs is using a mode like vim,  how to set emacs mode as default mode instead of key[i] or M-x->evil-mode into emacs mode?

2.When I edit golang file with using M-x->go-mode , I can't use space key as blank input the file, how to set it as normal blank input?

Macos
Emacs 26.2

Thank You.

**Comment by hlissner** [comment]
I am unable to understand either question; could you rephrase them?

**Comment by enko** [comment]
@yestab123 wrote:

> First open doom emacs is using a mode like vim, how to set emacs mode as default mode instead of key[i] or M-x->evil-mode into emacs mode?

Open your `~/.doom.d/init.el` and remove/comment out `(evil +everywhere)`, `evil-goggles` and change `(default +bindings +smartparens)` to `(default)` at the bottom of your `init.el`.

**Comment by flatwhatson** [comment]
As above, you should remove the `evil` lines from `init.el`, but I recommend leaving `(default +bindings +smartparens)`.  We have good out-of-the-box support for non-evil bindings now, and there are lots of nice commands under `C-c` provided by the default bindings.

**Comment by yestab123** [comment]
> As above, you should remove the `evil` lines from `init.el`, but I recommend leaving `(default +bindings +smartparens)`. We have good out-of-the-box support for non-evil bindings now, and there are lots of nice commands under `C-c` provided by the default bindings.

My English is not very good. It's work, Thanks for you help.

**Comment by yestab123** [comment]
> @yestab123 wrote:
> 
> > First open doom emacs is using a mode like vim, how to set emacs mode as default mode instead of key[i] or M-x->evil-mode into emacs mode?
> 
> Open your `~/.doom.d/init.el` and remove/comment out `(evil +everywhere)`, `evil-goggles` and change `(default +bindings +smartparens)` to `(default)` at the bottom of your `init.el`.

My English is not very good. It's work, Thanks for you help.
